kotlin-codegen/compiler/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.squareup.moshi</groupId>
+    <artifactId>moshi-parent</artifactId>
+    <version>1.6.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>moshi-kotlin-codegen-compiler</artifactId>
+
+  <repositories>
+    <repository>
+      <id>jcenter</id>
+      <url>https://jcenter.bintray.com/</url>
+    </repository>
```

Comment:
This will prevent us from releasing to Maven central.

Reply:
I hadn't thought of that. What do you suggest? I don't know if @Takhion has strong feelings on publishing to mavencentral (this is for kotlin-metadata)

Reply:
Quite the opposite! I've already received other requests to publish to Maven central, and the next version will be there for sure (soon) :+1:

Reply:
perf